isthmus/src/test/java/io/substrait/isthmus/TpcdsQueryNoValidation.java

```diff
@@ -2,11 +2,20 @@
 
 import com.google.protobuf.util.JsonFormat;
 import java.util.Arrays;
+import org.apache.calcite.adapter.tpcds.TpcdsSchema;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 
 public class TpcdsQueryNoValidation extends PlanTestBase {
 
+  private void testQuery(int i) throws Exception {
+    SqlToSubstrait s = new SqlToSubstrait();
+    TpcdsSchema schema = new TpcdsSchema(1.0);
```

Comment:
This is nice! maybe we should do the same for the TPCH schema instead of the [`schema.sql`](https://github.com/substrait-io/substrait-java/blob/main/isthmus/src/test/resources/tpch/schema.sql) we have currently.

Reply:
👍